src/containers/profile/ProfileSections/AccountSettings/AccountSettingsSections/ProfileInfo/ProfileInfo.jsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
```

Comment:
ok these directory paths are too deep. i just noticed this so sorry if it was like this in the other reviews, generally maher we never want more than 3 directories deep

so for this one, i would do

```
account/profile/ProfileInfo.jsx
```

maximum

Reply:
can we keep the path as it is and after the profile is done I'll make a PR for solving this issue for all profile components?

Reply:
please move it in this PR, the current path is kinda an unacceptable level of nesting for any type of code

in general, we don't commit wrong code and fix it later. when it gets committed, it needs to be correct

Reply:
to solve this PR I have to change a path also in a file that depends on this PR, I did that and requested a new PR called "Update path for deep components"